public/pattern-library/data.json

```diff
@@ -0,0 +1,128 @@
+{
+    "project_logo": null,
+    "project_favicon": "ryr_logo.png",
```

Comment:
I think the logo should also be set.

Reply:
I set the logo as the same image but I'm thinking we should change the favicon to the logo without the Request Yo Racks on the bottom.

Reply:
Yep, totally agree.